app/models/game.rb

```diff
@@ -61,4 +62,18 @@ def location_is_under_attack_by_color?(color, row, col)
 
     false
   end
+
+  def forfeit!(user)
+    player_ids = [black_player_id, white_player_id]
```

Comment:
instead of working with `ids`. I think it would be easier to use the object directly.

```Ruby
players = [black_player, white_player]
winner = players.find{ |player| player.id != forfeiting_user }
```